cstruct-async.opam

```diff
@@ -14,6 +14,7 @@ build: [
 ]
 depends: [
   "jbuilder" {build & >="1.0+beta7"}
+  "async"
```

Comment:
Cool. Fixed in `c194d88`.

Aside: why do we not mention Core in any of Jbuilder files? Is it because Jbuilder will transitively obtain dependencies? Even if that is the case, would it still be good documentation to include it since we are explicitly opening it in many of the files?